_episodes_rmd/14-tidyr.Rmd

```diff
@@ -129,6 +129,8 @@ We'll load the data file and look at it.  Note: we don't want our continent and
 country columns to be factors, so we use the stringsAsFactors argument for
 `read.csv()` to disable that.
 
+**** PROVIDE LINK TO gapminder_wide.csv HERE ****
+
```

Comment:
The link to the file is https://raw.githubusercontent.com/swcarpentry/r-novice-gapminder/gh-pages/_episodes_rmd/data/gapminder_wide.csv

It's in the data folder in the Rmd directory of the repo.